utilities/titandb/db_impl.cc

```diff
@@ -264,7 +284,7 @@ Status TitanDBImpl::GetImpl(const ReadOptions& options,
 
   BlobIndex index;
   s = index.DecodeFrom(value);
-  if (!s.ok()) return s;
+  assert(s.ok());
```

Comment:
This code will not run in the release build, and you should return error on corruption.